pkg/util/filesystem/permission_nowindows.go

```diff
@@ -54,3 +56,18 @@ func (p *Permission) RestrictAccessToUser(path string) error {
 
 	return nil
 }
+
+// RemoveAccessToOtherUsers on Unix this calls RestrictAccessToUser and then removes all access to the file for 'group'
+// and 'other'
+func (p *Permission) RemoveAccessToOtherUsers(path string) error {
```

Comment:
It's mostly related to the function above but why are we using a `Permission` here (I mean on `RestrictAccessToUser`). Is it a legacy code that has been changed and the argument `p *Permission` has never been removed ?

Reply:
It's for Windows. The Windows implementation need to pull a few IDs and ACL ( [here](https://github.com/DataDog/datadog-agent/blob/main/pkg/util/filesystem/permission_windows.go#L27) ).

Reply:
Ohhh, thanks 🙇